src/runtime_src/core/common/module_loader.h

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright (C) 2016-2020 Xilinx, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+#ifndef xrtcore_util_module_loader_h_
+#define xrtcore_util_module_loader_h_
+
+// This file contains a loader utility class for plugin modules
+//  that are loaded from either OpenCL or XRT level applications.
+
+#include <boost/filesystem/path.hpp>
```

Comment:
This include should be in the .cpp file.

Reply:
This include was moved into the cpp file along with the utility functions